changelog.txt

```diff
@@ -19,6 +19,7 @@ that repo.
 - `quickfort`: fix valid placement detection for floor hatches, floor grates, and floor bars (they were erroneously being rejected from open spaces and staircase tops)
 - `quickfort`: fix query blueprint statistics being added to the wrong metric when both a query and a zone blueprint are run by the same meta blueprint
 - `quickfort`: display missing blueprint labels in gui dialog list
+- `autounsuspend` and `unsuspend`: now both leave buildingplan-managed buildings alone
```

Comment:
They worked before the buildingplan changes, right? Unless they were broken in r3, this can probably be left out of the changelog.

Reply:
autounsuspend did, but unsuspend did not. updated changelog.

Reply:
Sure enough - thanks!